### ISSUE TITLE
Bookdown CSS

### ISSUE THREAD

**Issue by cmadland** [TWUOnline/bug-tracking#843]
**Which course contains the bug?**  
new bookdown sites
 
 
**Describe the error and/or your suggestion**  
Perhaps this is my misunderstanding but it seems the css for 2 'Learning Activity' and 'checking your learning' blocks is the same colour 
 
**Location of the bug**  
Please provide a specific link to the page with the error, as well as where the error is on the page:
1. URL:  https://sturdy-spork-wlnqeer.pages.github.io/
2. Unit #         Topic #         Activity#  
 
**Optional: Screenshots**  

![image](https://user-images.githubusercontent.com/12282929/232151570-f18a08d7-d928-4b13-99eb-975a682fcef8.png)

**Comment by Jeff-TWU** [comment]
@cmadland That is right. They are the same colour. I believe this is what we decided on. 

Happy to make changes if we need/want to. :) It's pretty easy now that our CSS is external.

**Comment by cmadland** [comment]
Maybe we consolidate the boxes on the index so it doesn't look like we made a mistake...

**Comment by Jeff-TWU** [comment]
> Maybe we consolidate the boxes on the index so it doesn't look like we made a mistake...

Sounds good to me. Below is my suggestion:

- We cut the _Learning Activities_ box.
- Change the _Checking Your Learning box_ description to: 

***Checking Your Learning*** 
This box will signify an ungraded assignment, discussion post, or a helpful list that can be used to check your understanding.

Thoughts? @kmarjanovic

**Comment by kmarjanovic** [comment]
Hi @Jeff-TWU & @cmadland.
I think the Learning Activities box is crucial.  Let's keep this green and with the current explanation.

The Checking Your Learning - I still prefer a different colour to differentiate it from the learning activity...or no box.  Its intention is to point back to the learning outcomes of the unit.  It is not necessarily a learning activity, so I'd rather go back to the purple.

Suggestion - Learning Activity (green) Assessment (blue), Checking Your Learning (either purple, grey - or with no box!); Note (grey or yellow?)  
Here are some notes: https://www.notion.so/twuid/Colour-Boxes-56c3ba4f4d5744dfb1e1b0c00959d529

Let's chat today about this

**Comment by Jeff-TWU** [comment]
@kmarjanovic @cmadland Can we please sort this out at the team meeting tomorrow?
